### PR TITLE
feat: grpc guide

### LINKED ISSUE #871: Write a guide to develop GraphQL endpoints on top of GRPC using Tailcall
Write a guide to use GRPC. Make sure it covers the feature in depth. Emphasize on how easy it is to use Tailcall to build a GraphQL API on top of GRPC. 

**Requirements**
- Make no grammatical or spelling errors.
- Keep the guide tone neutral, non non-emotional.
- Keep the writing style free-flowing and engaging.
- Fact-check everything, connect on Discord if you are confused.
- Make the writing cohesive, to the point, and related to the title.

**PS: DON'T CREATE LOW-EFFORT CHATGPT GENERATED CONTENT!**

### PR DESCRIPTION
**Pull request by meskill**
Closes https://github.com/tailcallhq/tailcall/issues/871

### CODE REVIEW

File: docs/guides/grpc.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+It not working example yet because we haven't specified how to use connect to server. To do this you may want to explore more about [`@grpc` operator](../operators/grpc.md). It's usage is pretty straightforward and require you to specify some required options like path to protobuf file itself, name of the service and method that should be used to make a call.
+
+If you need to provide any input to gRPC method call you can specify it with the `body` option that allows to specify a Mustache template and therefore it could use any input data like `args` and `value` to construct the body request. The body value is specified in the JSON format if you need to create the input manually and cannot just use `args` input.
+
+```graphql
+type Query {
+  news: NewsData! @grpc(service: "NewsService", method: "GetAllNews", protoPath: "news.proto")
+  newsById(news: NewsInput!): News!
+    @grpc(service: "NewsService", method: "GetNews", body: "{{args.news}}", protoPath: "news.proto")
```
Comment (wesleymatosdev) [inline docs/guides/grpc.md]: Need to change this and add the `@link(type: Protobuf)` to import the .proto file
```suggestion
  news: NewsData! @grpc(method: "news.NewsService.GetAllNews")
  newsById(news: NewsInput!): News!
    @grpc(method: "news.NewsService.GetNews", body: "{{args.news}}")
```

**Comment by tusharmath** [comment]
@meskill can you update this with the new link operator.

File: docs/guides/grpc.md (hunk 1 above)
Comment (meskill) [reply docs/guides/grpc.md]: updated

File: docs/guides/grpc.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+```graphql
+schema @server(port: 8000, graphiql: true) @upstream(baseURL: "http://localhost:50051", httpCache: true) {
+  query: Query
+}
+```
+
+To specify protobuf file to read types from, use `@link` operator with type `Protobuf` on schema. `id` is important part of the definition that will be used by `@grpc` operator later
+
+```graphql
+schema @link(id: "news", src: "./news.proto", type: Protobuf)
```
Comment (wesleymatosdev) [inline docs/guides/grpc.md:126]:
```suggestion
schema @link(id: "news-id", src: "./news.proto", type: Protobuf)
```

we could change it to differentiate between the link id and the package name when setting the `@grpc` directive

**Pull request merged**